lib/view_component/engine.rb

```diff
@@ -129,6 +129,13 @@ class Engine < Rails::Engine # :nodoc:
             internal: true
           )
 
+          get(
+            "view_components_system_test_entrypoint",
+            to: "#{preview_controller}#system_test_entrypoint",
+            as: :preview_view_component_entry_point,
```

Comment:
Why is this `as` defined? I think the default naming is likely more clear, as this isn't for Previews.

Reply:
Makes sense!